Makefile

```diff
@@ -27,6 +27,8 @@ endif
 GO_BUILD := CGO_ENABLED=0 go build -tags=$(BUILDTAGS) -ldflags=$(LDFLAGS)
 GO_TEST := CGO_ENABLED=0 go test -tags=$(BUILDTAGS) -ldflags=$(LDFLAGS)
 
+CNAB_BASE_INVOCATION_IMAGE_NAME := docker/cnab-app-base:$(BUILD_TAG)
```

Comment:
cf ,my comment about cleaning up `docker.Makefile`, either the equivalent needs to be removed there or this maybe needs to be in `vars.mk`?

Reply:
Done